lib/active_fedora/aggregation/builder.rb

```diff
@@ -5,8 +5,7 @@ class Builder < ActiveFedora::Associations::Builder::CollectionAssociation
     def build
       reflection = super
       configure_dependency
-      predicate = options.fetch(:predicate, ::RDF::Vocab::ORE.aggregates)
-      model.has_and_belongs_to_many :proxies, predicate: predicate, class_name: 'ActiveFedora::Aggregation::Proxy', inverse_of: :containers
+      model.has_and_belongs_to_many :proxies, predicate: RDF::URI("http://test.com/hasProxy"), class_name: 'ActiveFedora::Aggregation::Proxy', inverse_of: :containers
```

Comment:
We need a real predicate for this. This is just kind of an implementation detail at this point - maybe replaceable with something which queries down Contains?